resources/assets/less/bem/user-card-brick.less

```
@@ -4,18 +4,17 @@
 .user-card-brick {
   @_top: user-card-brick;
   @_fade-duration: 220ms;
-
-  @_background-color: @osu-colour-b6;
   @_border-radius: @border-radius-large;
 
   --border: transparent;
 
-  margin: var(--item-margin);
-  flex: none;
+  .link-white();
+  .link-plain();
 
+  margin: var(--item-margin);
   width: min-content;
   max-width: 100%;
-  background-color: @_background-color;
+  background-color: osu-hsla(@osu-colour-b4, 0.8);
```

Don't use `osu-hsla` anymore.

```suggestion
  background-color: hsla(var(--hsl-b4), 0.8);
```